### PR TITLE
MultilineInput to core

### PR DESCRIPTION
**Pull request by libertymayc**
Multiline to core

### CODE REVIEW

File: packages/core/stories/multiline-input/multiline-input.stories.tsx [hunk 1]
```diff
@@ -1,23 +1,20 @@
-import { Button, FlowLayout, Label, Text } from "@salt-ds/core";
-import { MultilineInput } from "@salt-ds/lab";
+import { ChangeEvent, Label, useState } from "react";
```
Comment (alycrys) [inline packages/core/stories/multiline-input/multiline-input.stories.tsx:1]: Why are you importing Label from react? :)

Comment (libertymayc) [reply packages/core/stories/multiline-input/multiline-input.stories.tsx:1]: hehe

File: packages/core/src/multiline-input/MultilineInput.tsx [hunk 2]
```diff
@@ -9,14 +9,11 @@ import {
...
-  useControlled,
-  useFormFieldProps,
-} from "@salt-ds/core";
 import { useComponentCssInjection } from "@salt-ds/styles";
 import { useWindow } from "@salt-ds/window";
+import { useFormFieldProps } from "../form-field-context";
+import { StatusAdornment } from "../status-adornment";
+import { makePrefixer, useControlled } from "../utils";
 
 import multilineInputCss from "./MultilineInput.css";
```
Comment (joshwooding) [inline packages/core/src/multiline-input/MultilineInput.tsx:19]: Nit: you're picking from "input" but it probably should be "textarea" although nothing should change

File: .changeset/eighty-jars-explain.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+"@salt-ds/core": minor
+"@salt-ds/lab": minor
+---
+
+Moved `MultilineInput` from lab to core
```
Comment (joshwooding) [inline .changeset/eighty-jars-explain.md:6]: Can you align this to how the other changeset have been done please e.g. Accordion

Comment (libertymayc) [reply .changeset/eighty-jars-explain.md]: done

File: packages/core/stories/input/input.stories.tsx [hunk 4]
```diff
@@ -28,6 +28,7 @@ export const Controlled: ComponentStory<typeof Input> = (args) => {
   const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
     const value = event.target.value;
     setValue(value);
+    console.log('changing')
```
Comment (joshwooding) [inline packages/core/stories/input/input.stories.tsx:31]: I guess this will be removed?

Comment (libertymayc) [inline packages/core/stories/input/input.stories.tsx:31]:
```suggestion
```

Comment (libertymayc) [inline packages/core/stories/input/input.stories.tsx:31]:
```suggestion
```

**Review by alycrys — APPROVED** [review]

**Review by joshwooding — APPROVED** [review]

**Pull request merged**